components/Shared/Author.js

```diff
@@ -0,0 +1,269 @@
+                <div className="flex py-2 justify-between my-2 items-center">
+                  <p className="text-base text-black-500 dark:text-white-500">
+                    Verse Commentary Source
+                  </p>
+                  <Switch
+                    checked={isVerseCommentarySourceEnabled}
+                    onChange={setIsVerseCommentarySourceEnabled}
+                    className={`${
+                      isVerseCommentarySourceEnabled
+                        ? "bg-my-orange"
+                        : "bg-gray-300"
+                    }
+                    relative inline-flex flex-shrink-0 h-[29px] w-[49px] border-2 border-transparent rounded-full cursor-pointer transition-colors ease-in-out duration-200 focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-75`}
+                  >
+                    <span
+                      aria-hidden="true"
+                      className={`${
+                        isVerseCommentarySourceEnabled
+                          ? "translate-x-5"
+                          : "translate-x-0"
+                      }
+                      pointer-events-none inline-block h-[25px] w-[25px] rounded-full bg-white shadow-lg transform ring-0 transition ease-in-out duration-200`}
+                    />
+                  </Switch>
+                </div>
+
+                <div className="flex py-2 justify-between items-center mb-2">
+                  <p className="text-base text-black-500 dark:text-white-500">
+                    Verse Transliteration Language
+                  </p>
+                  <Switch
+                    checked={isVerseTransliterationLanguageEnabled}
+                    onChange={setIsVerseTransliterationLanguageEnabled}
+                    className={`${
+                      isVerseTransliterationLanguageEnabled
+                        ? "bg-my-orange"
+                        : "bg-gray-300"
+                    }
+                    relative inline-flex flex-shrink-0 h-[29px] w-[49px] border-2 border-transparent rounded-full cursor-pointer transition-colors ease-in-out duration-200 focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-75`}
+                  >
+                    <span
+                      aria-hidden="true"
+                      className={`${
+                        isVerseTransliterationLanguageEnabled
+                          ? "translate-x-5"
+                          : "translate-x-0"
+                      }
+                      pointer-events-none inline-block h-[25px] w-[25px] rounded-full bg-white shadow-lg transform ring-0 transition ease-in-out duration-200`}
+                    />
+                  </Switch>
+                </div>
+
+                <div
+                  class="mb-6"
+                  hidden={!isVerseTransliterationLanguageEnabled}
+                >
+                  <Listbox
```

Comment:
I am not able to see `ListBox` on the website. Please check.
![image](https://user-images.githubusercontent.com/11087313/151209325-940073c8-299e-4b46-aa34-a827d1fd57dd.png)

Reply:
You get to see that whenever toggle is on!

Reply:
oh, ok. There is an issue in the dark mode.
![image](https://user-images.githubusercontent.com/11087313/151501558-b5e48c1e-ba3e-4683-8800-c069cab04980.png)